### PR TITLE
Brightcrowd Integration

### PR DESCRIPTION
**Pull request by travistatro**
## Describe your changes
1. Added BrightCrowd integration to Nango, including:
2. Documentation file brightcrowd.mdx with API description, key features, endpoints, authentication method, use cases, and resources.
3. API configuration in providers.yaml
4. SVG logo for BrightCrowd

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
[-] I added tests, otherwise the reason is: This PR is only adding documentation for a new API integration
[-] I added observability, otherwise the reason is: Not applicable for API documentation
[-] I added analytics, otherwise the reason is: Not necessary for adding a new API integration documentation
The main changes are in the following file:

### CODE REVIEW

File: docs-v2/integrations/all/brightcrowd.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (bodinsamuel) [inline docs-v2/integrations/all/brightcrowd.mdx:1]: If you can follow other documentation models that would be great. No need to fill information if you are not sure

File: packages/shared/providers.yaml [hunk 2]
```diff
@@ -787,6 +787,27 @@ calendly:
         - owner
     docs: https://docs.nango.dev/integrations/all/calendly
 
+brightcrowd:
+    display_name: BrightCrowd
+    categories:
+        - alumni-data
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:793]:
```suggestion
        - social
```

Comment (bodinsamuel) [reply packages/shared/providers.yaml:793]: Can you also add it to social.mdx please

File: packages/shared/providers.yaml [hunk 3]
```diff
@@ -787,6 +787,27 @@ calendly:
...
+    auth_mode: BASIC
+    token_url: https://bcb-staging.auth.us-east-1.amazoncognito.com/oauth2/token
+    token_request_auth_method: basic
+    token_params:
+        grant_type: client_credentials
+        scope: bcb.partner/book.read
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:799]: Not sure it's correct, you have to request this scope in your own integration no?

File: packages/shared/providers.yaml [hunk 4]
```diff
@@ -787,6 +787,27 @@ calendly:
         - owner
     docs: https://docs.nango.dev/integrations/all/calendly
 
+brightcrowd:
+    display_name: BrightCrowd
+    categories:
+        - alumni-data
+    auth_mode: BASIC
+    token_url: https://bcb-staging.auth.us-east-1.amazoncognito.com/oauth2/token
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:795]: Is staging correct? 
they don't advertise any API on their website, not confident about this URL

**Review by bodinsamuel — COMMENTED** [review]
Thanks for your contribution. It would be great to share the documentation if you have some so we can double check the values there

**Comment by hassan254-prog** [comment]
@travistatro , just to add on what @bodinsamuel has mentioned, according to the provider's documentation, the authentication mode should be`OAUTH2_CC`

**Comment by hassan254-prog** [comment]
> @travistatro , just to add on what @bodinsamuel has mentioned, according to the provider's documentation, the authentication mode should be`OAUTH2_CC`

Probably this would work but will have to confirm by creating a new connection on Nango and also try on the pagination especially `limit_name_in_request` as it hasn't been mentioned in the docs :
```
brightcrowd:
    display_name: BrightCrowd
    categories:
        - social
    auth_mode: OAUTH2_CC
    token_url: https://bcb-staging.auth.us-east-1.amazoncognito.com/oauth2/token
    token_request_auth_method: basic
    token_params:
        grant_type: client_credentials
    proxy:
        base_url: https://api.brightcrowd.com/partner
        headers:
            Accept: application/problem+json
        paginate:
            type: cursor
            cursor_path_in_response: nextPageToken
            cursor_name_in_request: pageToken
            limit_name_in_request: limit
    docs: https://docs.nango.dev/integrations/all/brightcrowd

**Comment by travistatro** [comment]
> Thanks for your contribution. It would be great to share the documentation if you have some so we can double check the values there

Thank you for the feedback/responses. I am sharing this with an engineer on my team to help move some of this forward!

**Comment by hassan254-prog** [comment]
@travistatro , I have opened a new [PR](https://github.com/NangoHQ/nango/pull/2962) in reference to this. Could you be having test credentials to test the implementation?

**Comment by TBonnin** [comment]
@hassan254-prog is this PR still relevant?

**Comment by hassan254-prog** [comment]
> @hassan254-prog is this PR still relevant?

I think we can close since we already added support for `Brightcrowd` in [here](https://github.com/NangoHQ/nango/pull/2962).

**Pull request closed without merging**